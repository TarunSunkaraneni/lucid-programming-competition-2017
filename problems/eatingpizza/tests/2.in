10
150 207 217
992 131 440
438 268 405
987 383 463
14 96 154
200 196 210
347 133 486
828 226 359
30 59 486
52 167 410
