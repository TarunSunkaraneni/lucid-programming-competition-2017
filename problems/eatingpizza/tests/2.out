Jacob
Megan
Megan
Jacob
Jacob
Megan
Megan
Megan
Jacob
Jacob
